packages/protocol/contracts-0.8/governance/Validators.sol

```diff
@@ -713,17 +713,23 @@ contract Validators is
    * @param account The address of the validator group.
    * @param n The number of members to return.
    * @return The top n group members for a particular group.
+   * @dev Returns the account instead of signer on L2.
    */
   function getTopGroupValidators(
     address account,
     uint256 n
   ) external view returns (address[] memory) {
     address[] memory topAccounts = groups[account].members.headN(n);
     address[] memory topValidators = new address[](n);
-    for (uint256 i = 0; i < n; i = i.add(1)) {
-      topValidators[i] = getAccounts().getValidatorSigner(topAccounts[i]);
+
+    if (isL2()) {
+      return topAccounts;
+    } else {
+      for (uint256 i = 0; i < n; i = i.add(1)) {
```

Comment:
would add a note pointing to an issue that should be in be post-mainnet milestone